internal/workspace_test.go

```diff
@@ -76,6 +76,18 @@ func TestAccWorkspaceIntegration(t *testing.T) {
 
 func TestAccWorkspaceRecursiveListNoTranspile(t *testing.T) {
 	ctx, w := workspaceTest(t)
+	notebook := myNotebookPath(t, w)
+
+	// Import the test notebook
+	err := w.Workspace.Import(ctx, workspace.Import{
+		Path:      notebook,
+		Format:    "SOURCE",
+		Language:  "PYTHON",
```

Comment:
There are constants for these values like `workspace.LanguagePython` and `workspace. ExportFormatSource `, I'd better use them

Reply:
Good point -- I copied this code from elsewhere in the file to unblock.